composer.json

```diff
@@ -22,7 +22,7 @@
     },
     "require": {
         "php": "^7.2 || ^8.0",
-        "aeviiq/collection": "^3.1",
+        "aeviiq/collection": "^4.0",
```

Comment:
In the version 4.x the support for php 8.0 and below are dropped. So @nusje2000 can you consider to drop php 7.x?

Reply:
I don't really like the fact that we are limited to 8.0 just because of a utility package for managing arrays, but it would require a major refactor to fix this. So I'm fine with this change as 7.0 can still use an earlier release of this package.

@LucasdeRijke could you remove 7.2 from this composer file, and change the 8.0 to 8.1?

Reply:
Can't you make a 3.0 version for this package and drop support for PHP < 8.1, as those version are already EOL?

Reply:
Making a 3.0 release is also an option, but I would opt for changing the composer.json to this:
```
        "php": "^7.2 || ^8.0",
        "aeviiq/collection": "^3.0 || ^4.0",
```

This package can use both version of the collection package (as this PR only changed the version requirement, and not any of the actual code) and this would allow the package to be used on php 7 and 8. If we can I would like to prevent making a breaking change, and this seems like that should be possible without any effort in this case.